### ISSUE TITLE
Incorrect dimensions for video recorded from iOS Simulator

### ISSUE THREAD

**Issue by sindresorhus** [sindresorhus/gifski-app#76]
I've seen this issue in Crashlytics, but couldn't reproduce until I got this reply: https://twitter.com/FlorianBuerger/status/1124945762729779200

I've now managed to reproduce the issue by running an iOS app in a simulator like `iPhone XS` and then running `xcrun simctl io booted recordVideo simulator-recording.mp4` in the terminal, and then pressing Ctrl+C some seconds later to end the recording.

Here's a test file: [ios-simulator-recording-incorrect-dimensions-fixture.mp4.zip](https://github.com/sindresorhus/gifski-app/files/3145403/ios-simulator-recording-incorrect-dimensions-fixture.mp4.zip)

And here's the debug output when dragging it into Gifski:

```
## AVAsset debug info ##
Extension: mp4
Video codec: H264 (avc1)
Audio codec: nil
Duration: 7s
Dimension: 0×0
Frame rate: 14.99
File size: 3.7 MB
Is readable: true
Is playable: true
Is exportable: true
Has protected content: false

Track #1
----
Type: Video
Codec: H264 (avc1)
Duration: 7s
Dimensions: 0×0
Natural size: (1125.0, 0.03717041015625)
Frame rate: 14.99
Is playable: true
Is decodable: true
----
```

Notice `Natural size`, which is the [`AVAssetTrack#naturalSize` property](https://developer.apple.com/documentation/avfoundation/avassettrack/1387724-naturalsize), and should be the dimensions.

I have no idea why the height is so screwed up. Seems to me that this is a macOS bug, unless I'm missing something?

**Comment by florianbuerger** [comment]
I was about to mention that workaround but you were faster 🙂 Glad I could help to reproduce it.

**Comment by sindresorhus** [comment]
@florianbuerger Does it look like an AVFoundation bug to you too? Considering whether I should open a Radar about this.

**Comment by florianbuerger** [comment]
Yes it does. Not sure wether the bug is in reading in `AVAssetTrack` or in writing it in the first place (I guess via `AVAssetExportSession` when the video is recorded).

**Comment by sindresorhus** [comment]
No longer seeing this.
